asv_bench/benchmarks/benchmarks.py

```diff
@@ -0,0 +1,136 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import modin.pandas as pd
+from modin.config import TestDatasetSize
+from .utils import generate_dataframe, RAND_LOW, RAND_HIGH
+
+pd.DEFAULT_NPARTITIONS = 4
```

Comment:
It would be nice to somehow make `pd.DEFAULT_NPARTITIONS` a parameter to be able to run same benchmarks with different settings. This way we could see how they scale on different number of CPU cores.

Reply:
I agree with you.

In order not to generate new modin environment variables - I suppose we can use `MODIN_CPUS` to adjust the number of partitions. How do you like this solution?